tools/npi-csv

```diff
@@ -3,4 +3,4 @@
 set -e
 set -o pipefail
 
-awk -F, '{ printf "%s,%s\n", $1, $2 }' | tr -d '"'
\ No newline at end of file
+awk -F'\t' '{ if($2 != "\\N"){printf "%s,%s\n", $1, $2;}}' | tr -d '"'
```

Comment:
$0.02: did it switch from csv to tsv?

Reply:
the npis-latest-dump.csv file is tab separated on s3.